tests/runner.py

```diff
@@ -1209,7 +1216,12 @@ def load_test_suites(args, modules):
       except AttributeError:
         pass
     if len(names_in_module) > 0:
-      suites.append((m.__name__, loader.loadTestsFromNames(sorted(names_in_module), m)))
+      tests = loader.loadTestsFromNames(sorted(names_in_module), m)
+      suite = suite_for_module(m)
```

Comment:
so does this mean that parallel is now the only way to run? How much extra code/support would it take to keep the sequential way?